nest-city-account/src/utils/subservices/cognito.subservice.ts

```diff
@@ -235,17 +236,29 @@ export class CognitoSubservice {
 
   /**
    * Returns all users from cognito user pool
-   * @returns UsersListType
+   * @returns CognitoGetUserData[]
    */
-  async getAllCognitoUsers(): Promise<UsersListType> {
-    let result: UsersListType = []
+  async getAllCognitoUsers(): Promise<CognitoGetUserData[]> {
+    let result: CognitoGetUserData[] = []
     const params: ListUsersRequest = {
       UserPoolId: this.config.cognitoUserPoolId,
     }
     do {
-      const { Users = [], PaginationToken } = await this.cognitoIdentity.listUsers(params).promise()
-      result = [...result, ...Users]
-      params.PaginationToken = PaginationToken
+      // const { Users = [], PaginationToken } = await this.cognitoIdentity.listUsers(params).promise()
```

Comment:
This comment should not be here if there is a new implementation already. (now we will likely revert this change anyway, just for future reference)